docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -428,12 +427,12 @@ _Spoiler: Maintained by us._
 
 An Open Source development framework that enables you to easily add a "Secure by Default" architecture to your existing
 software with only a few lines of code. It's loosely based on the OWASP Application Security Verification Standard 
-(above) and is also designed to be implemented in [multiple levels](https://www.lunasec.io/docs/pages/how-it-works/security/introduction/).
+(above), and is also designed to be implemented at [multiple levels](https://www.lunasec.io/docs/pages/how-it-works/security/introduction/).
```

Comment:
I just don't like including this "multiple levels" stuff here at all. It's an implementation detail of our framework, not an interesting selling point or salient detail. Basically, it doesn't convey anything useful to a first-time reader and we could be using this space for something better.  

If we want to say this in a more meaningful way, maybe we could say "adopted gradually" or something

Reply:
Hm.  Okay, it sounds like the person writing this was trying to make it sound like a selling point, but altered as suggested.